includes/talks/tags.php

```diff
@@ -455,40 +455,6 @@ function wct_talks_get_author_id() {
 }
 
 /**
- * Checks if the Talk being iterated on is sticky
- *
- * @package WordCamp Talks
- * @subpackage talks/tags
- *
- * @since 1.0.0
- *
- * @return bool True if the Talk being iterating on is sticky, false otherwise
- */
-function wct_talks_is_sticky_talk() {
```

Comment:
As you're removing this function, we need to also remove its use from the `wct_talks_get_before_talk_title()` function.

```diff
diff --git includes/talks/tags.php includes/talks/tags.php
index 2cb454f..c960a91 100644
--- includes/talks/tags.php
+++ includes/talks/tags.php
@@ -591,10 +553,6 @@ function wct_talks_before_talk_title() {
 	function wct_talks_get_before_talk_title() {
 		$output = '';
 
-		if ( wct_talks_is_sticky_talk() ) {
-			$output = '<span class="sticky-talk"></span> ';
-		}
-